games-fps/gzdoom/metadata.xml

```diff
@@ -14,6 +14,7 @@
 		<name>Gentoo Games Project</name>
 	</maintainer>
 	<use>
+		<flag name="debug">Enable assert() functions for debugging</flag>
```

Comment:
This description matches the global description of `debug`, so we probably don't need this line added here.